content/docs/codex/codex.yaml

```diff
@@ -0,0 +1,227 @@
+        headers:
+          type: array
+          items:
+            type: string
+        include_spans:
+          type: boolean
+          description: IncludeSpans (Deprecated) If the timing values should be included in the response.
+          deprecated: true
+        metadata:
+          type: object
+          additionalProperties:
+            type: string
+          example: {"/boot-time": "1542834188","/last-reconnect-reason": "spanish inquisition" }
+        msg_type:
+          type: integer
+          format: int64
+          example: 4
+        partner_ids:
+          description: the partner ids used by the device. Determined from the same event that provides the state
+          type: array
+          items:
+            type: string
+          example: ["example_partner", ".*"]
+        payload:
+          type: array
+          items:
+            type: integer
+            format: uint8
+          example: eyJpZCI6IjUiLCJ0cyI6IjIwMTktMDItMTJUMTE6MTA6MDIuNjE0MTkxNzM1WiIsImJ5dGVzLXNlbnQiOjAsIm1lc3NhZ2VzLXNlbnQiOjEsImJ5dGVzLXJlY2VpdmVkIjowLCJtZXNzYWdlcy1yZWNlaXZlZCI6MH0=
+        rdr:
+          description: RequestDeliveryResponse The request delivery response is the delivery result of the previous (implied request) message with a matching transaction_uuid
+          type: integer
+          format: int64
+        source:
+          description: the originating descriptoin of the request or response.
+          type: string
+          example: dns:talaria.xmidt.example.com
+        spans:
+          description: "Spans An array of arrays of timing values as a list in the format: \"parent\" (string), \"name\" (string),\n\"start time\" (int), \"duration\" (int), \"status\" (int)"
+          type: array
+          items:
+            type: array
+            items:
+              type: string
+        status:
+          description: The response status from the originating service.
+          type: integer
+          format: int64
+        transaction_uuid:
+          description: The transaction key for the message
+          type: string
+          example: 546514d4-9cb6-41c9-88ca-ccd4c130c525
+        url:
+          description: The url to use when connecting to the nanomsg pipeline
+          type: string
+      example:
+        {
+          "msg_type": 4,
+          "source": "dns:talaria.xmidt.example.com",
+          "dest": "event:device-status/event:device-status/mac:ffffffffdae4/online/online",
+          "transaction_uuid": "546514d4-9cb6-41c9-88ca-ccd4c130c525",
+          "content_type": "json",
+          "metadata": {
+            "/boot-time": "1542834188",
+            "/last-reconnect-reason": "spanish inquisition"
+          },
+          "payload": "eyJpZCI6IjUiLCJ0cyI6IjIwMTktMDItMTJUMTE6MTA6MDIuNjE0MTkxNzM1WiIsImJ5dGVzLXNlbnQiOjAsIm1lc3NhZ2VzLXNlbnQiOjEsImJ5dGVzLXJlY2VpdmVkIjowLCJtZXNzYWdlcy1yZWNlaXZlZCI6MH0=",
+          "birth_date": 1580326503649365642
+        }
+
+
+    Status:
```

Comment:
This is the only place where the descriptions aren't even close to full sentences, so I didn't add periods.  Not sure if we want to fix this or just leave it.  It's not consistent with the rest of the page, but it is consistent within itself, so that's good at least.